tutor/src/models/course.js

```diff
@@ -215,8 +215,8 @@ class Course extends BaseModel {
     return Boolean(this.does_cost && this.userStudentRecord && !this.userStudentRecord.isUnPaid);
   }
 
-  @computed get defaultTimes() {
-    return pick(this, 'default_due_time', 'default_open_time');
+  momentInZone(date) {
+    return moment(date).tz(this.time_zone);
```

Comment:
Does this convert the date or just append the timezone to a timezone-less date?
I ask because the BE handles opens_at and due_at as a special case and they are supposed to be treated as time-zone-less in the FE (they are always in whatever time zone the course is, but without any conversion).

The BE will completely ignore any timezone you send for opens_at, due_at.

Reply:
no it converts to the course's timezone.  We have to do that so we can send adjust the dates time in the course timezone and also send to the server in that timezone.

Reply:
From our chat, this needs to be a single call to tz to be correct, something like `moment.tz(date, this.time_zone)`

Reply:
yep, I made the change